examples/null.md

```diff
@@ -0,0 +1,141 @@
+# Null primitive type
+
+The `nullable` type attribute can be used to outline that `null` is an accepted value.
```

Comment:
I'd avoid using `null` in code span – to not create the impression that there may be something like `null` value. I'd suggest to use "unset" (as above).